src/jsonld/event.tsx

```diff
@@ -18,6 +27,9 @@ export interface EventJsonLdProps extends JsonLdProps {
   offers?: Offers | Offers[];
   aggregateOffer?: AggregateOffer;
   performers?: Performer | Performer[];
+  organizer?: Organizer;
+  eventStatus?: `https://schema.org/${EventStatus}`;
+  eventAttendanceMode?: `https://schema.org/${EventAttendanceMode}`;
```

Comment:
Let's update these so we don't need to pass in the entire URL. 

So instead of:

```
 eventStatus="https://schema.org/EventScheduled"
```

it would be:

```
EventScheduled
```